text/0096-auto-batching.md

```diff
@@ -83,6 +60,6 @@ N/A
 
 - Extends it for all consecutive payload types.
 - Add a filter capability by `batchUid` on the `/tasks` endpoints.
-- Do not fail the entire transaction if a document is not valid. Report the documents that could not be indexed to the user.
-- Enable auto-batching by default.
-- Optimize some tasks sequence, for example if there is a document addition followed by an index deletion, we could skip the document addition
\ No newline at end of file
+- <s>Do not fail the entire transaction if a document is not valid. Report the documents that could not be indexed to the user.</s>
+- <s>Enable auto-batching by default.</s>
```

Comment:
```suggestion
```
Ps: Integrate this suggestion given our final decision regarding the discussion in the previous comment about the format